<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QLabel" name="label_slice">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_stack">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <widget class="QSlider" name="slider_slice">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0">
     <widget class="QSlider" name="slider_stack">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="MplWidget" name="MplWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>400</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>682</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="menu_load"/>
    <addaction name="menu_add"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="menu_pca_2"/>
    <addaction name="menu_labeling"/>
    <addaction name="menu_clustering"/>
    <addaction name="separator"/>
    <addaction name="actionSVM_2"/>
   </widget>
   <widget class="QMenu" name="menuProjection">
    <property name="title">
     <string>Projection</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionCoronal"/>
    <addaction name="actionSagittal"/>
    <addaction name="actionTransversal"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menu_mask_2">
     <property name="title">
      <string>Label Mask</string>
     </property>
     <addaction name="menu_mask_positive"/>
     <addaction name="menu_mask_negative"/>
     <addaction name="menu_mask_none"/>
    </widget>
    <addaction name="menu_mri"/>
    <addaction name="menu_segmented"/>
    <addaction name="menu_windowing"/>
    <addaction name="menu_mask_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProjection"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menu_add">
   <property name="text">
    <string>Add Box</string>
   </property>
  </action>
  <action name="menu_labeling">
   <property name="text">
    <string>Labeling</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionCoronal">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Coronal</string>
   </property>
  </action>
  <action name="actionSagittal">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Sagittal</string>
   </property>
  </action>
  <action name="actionTransversal">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Transversal</string>
   </property>
  </action>
  <action name="menu_load">
   <property name="text">
    <string>Load Data</string>
   </property>
  </action>
  <action name="menu_pca">
   <property name="text">
    <string>PCA</string>
   </property>
  </action>
  <action name="actionSVM">
   <property name="text">
    <string>SVM</string>
   </property>
  </action>
  <action name="menu_clustering">
   <property name="text">
    <string>Clustering</string>
   </property>
  </action>
  <action name="menu_mri">
   <property name="text">
    <string>MRI</string>
   </property>
  </action>
  <action name="menu_segmented">
   <property name="text">
    <string>Segmented</string>
   </property>
  </action>
  <action name="actionSVM_2">
   <property name="text">
    <string>SVM</string>
   </property>
  </action>
  <action name="menu_windowing">
   <property name="text">
    <string>Windowing</string>
   </property>
  </action>
  <action name="menu_pca_2">
   <property name="text">
    <string>PCA 2</string>
   </property>
  </action>
  <action name="menu_mask_positive">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>positive</string>
   </property>
  </action>
  <action name="menu_mask_negative">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>negative</string>
   </property>
  </action>
  <action name="menu_mask_none">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>none</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
